docs/data/base/components/number-input/number-input.md

```diff
@@ -91,6 +91,82 @@ The following code snippet:
 />
 ```
 
+### Events
+
+The NumberInput component provides two props - `onChange` and `onInputChange` - that accept event handlers for when the value of the component changes.
```

Comment:
```suggestion
The Number Input component provides two props—`onChange` and `onInputChange`—that accept event handlers for when the value of the component changes.
```

Reply:
Ah I always forget this one, wonder if `markdownlint` can detect it

Reply:
ooh yeah it would be nice to have a rule to enforce it!

Reply:
You can't really see the change in monospaced GH text, but be sure to change these en-dashes ("-") to em-dashes ("—") in addition to removing the spaces.